source/features/faster-pr-diff-options.tsx

```diff
@@ -72,6 +72,12 @@ function init(): false | void {
 		)
 	);
 
+	// Trim title
+	const prTitle = select('.pr-toolbar .js-issue-title');
+	if (prTitle && select.exists('.pr-toolbar progress-bar')) { // Only review view has progress-bar
+		prTitle.style.maxWidth = '24em';
```

Comment:
That said, we could also set the title attribute for a tooltip with the full PR title on hover.
```suggestion
		prTitle.style.maxWidth = '24em';
		prTitle.title = prTitle.textContent;
```